### PR TITLE
Added Bootstrap 4

### PR DESCRIPTION
**Pull request by cibily**
In order to style in a easy way the app I added Bootstrap 4.

### CODE REVIEW

File: assets/styles/importer.less [hunk 1]
```diff
@@ -28,3 +28,8 @@
 // @import 'pages/signup.less';
 //
 // etc.
+/*
+@import 'bootstrap/less/bootstrap.less';*/
+@import 'custom.less';
```
Comment (marcopeg) [inline assets/styles/importer.less]: why to import "custom.less" if bootstrap is the only thing you want to use? 
this is a bit overengineering the file structure.
keep it simple and include bootstrap's components directly here.

File: views/dress/today.ejs [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
-<% listOfDresses.forEach(function(dress) { %>
-<div>
-  <h3><%= dress.id %> - <%= dress.name %></h3>
-  <h4><%= dress.season %></h4>
-  <img src="/images/<%= dress.picture %>" width=200>
 
+<div class="container">
+    <div class="row">
+        <div class="col-md-12">
+            <% listOfDresses.forEach(function(dress) { %>
```
Comment (marcopeg) [inline views/dress/today.ejs]: this is the point in which you repeat a piece of HTML for each dress you have in the list.

I expect you to use one row per item but I see that both row and column are outside the loop. 
how comes?

File: views/layout.ejs [hunk 3]
```diff
@@ -1,94 +1,92 @@
 <!DOCTYPE html>
```
Comment (marcopeg) [inline views/layout.ejs]: eventually you did zero modifications to this file but you included it in your commits anyway!
why???

be careful of unstaging files that are not meant to be committed

Comment (cibily) [inline views/layout.ejs]: ok. I understand! :-)

**Pull request merged**